src/Container.ts

```diff
@@ -70,7 +70,7 @@ export class Container {
         let service = this.findService(identifier);
 
         // find if instance of this object already initialized in the container and return it if it is
-        if (service && service.instance !== null)
+        if (service && service.instance !== null && service.instance !== undefined)
```

Comment:
why don't symply `service.instance != null`?

Reply:
because of strict checking rule